test/post-911-gib-status/containers/StatusPage.unit.spec.jsx

```diff
@@ -28,13 +33,13 @@ const teardown = () => {
 describe('<StatusPage>', () => {
   beforeEach(setup);
   it('should render', () => {
-    const tree = SkinDeep.shallowRender(<StatusPage store={store}/>);
+    const tree = SkinDeep.shallowRender(<StatusPage store={store} {...defaultProps}/>);
     const vdom = tree.getRenderOutput();
     expect(vdom).to.exist;
   });
 
-  it('should show title and print button', () => {
-    const node = findDOMNode(ReactTestUtils.renderIntoDocument(<StatusPage store={store}/>));
+  it.only('should show title and print button', () => {
```

Comment:
This test is failing. Seems to be because when the component is rendered, `enrollmentData` is `null`, even though I specifically pass it with the `veteranIsEligible` value passed to it. I'm stumped as to why it's rendering as null :/